src/util/floatbv_expr.cpp

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************\
+
+Module: API to expression classes for floating-point arithmetic
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+#include "floatbv_expr.h"
+
+#include "arith_tools.h"
+#include "bitvector_types.h"
+
+constant_exprt floatbv_rounding_mode(unsigned rm)
+{
+  // the 32 bits are an arbitrary choice
+  return ::from_integer(rm, unsignedbv_typet(32));
```

Comment:
Shouldn't the comment also clarify that the type needs to be the same as that of the `__CPROVER_rounding_mode` symbol (which, actually, seems to be a signed int)?